### PR TITLE
WIP

### LINKED ISSUE #9: [FEATURE] Color theme
### Type of feature

An improvement to something that already exists

### The name of the feature

Apply the color theme for the Flutter mobile app

### Feature description

- [ ] Ensure the color theme for the mobile app is correctly applied everywhere

### Additional information

- Primary : `Color(0xFFFCE92A)`
- Secondary : `Color(0xFF020105)`

### PR DESCRIPTION
**Pull request by LeoBruant**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Closes #9

## Type of change

Please add any relevant information to the type of change you've made.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update (or is a documentation update)
-   [ ] HIGH IMPORTANCE (requires a fast merging)

## Has This Been Tested?

Please describe the tests that you ran to verify your changes. It is way more likely for a PR to be merged and approved with successful tests.

-   [x] Yes, I ran tests
-   [ ] No, I did not include any test
-   [ ] No test is needed for this change

### CODE REVIEW

File: mobile/lib/widgets/event/detail.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+              const Text('Some text'),
+              const Spacer(),
+              GestureDetector(
+                child: const Text(
+                  'view planet',
+                  style: TextStyle(
+                      // color: Colors.blue,
+                      ),
+                ),
+                onTap: () {},
```
Comment (Spacelocust) [inline mobile/lib/widgets/event/detail.dart]:
```suggestion
```

File: compose.yml [hunk 2]
```diff
@@ -57,3 +57,17 @@ services:
     depends_on:
       - api
       - postgres
+
+  # Uncomment to use Flutter
```
Comment (YummYume) [inline compose.yml]: Mettre dans le README à la place

File: mobile/lib/main.dart [hunk 3]
```diff
@@ -72,19 +72,17 @@ class ForDemocracyApp extends StatelessWidget {
...
-        secondaryHeaderColor: ThemeColors.secondary,
-        colorScheme: ColorScheme.fromSwatch(
-          primarySwatch: ThemeColors.primaryMaterial,
-          accentColor: ThemeColors.secondary,
+        colorScheme: ColorScheme.dark(
+          primary: ThemeColors.primary,
+          secondary: ThemeColors.primary,
+          // background: ThemeColors.surface,
         ),
-        useMaterial3: true,
```
Comment (YummYume) [inline mobile/lib/main.dart:102]: Remettre ?

File: mobile/lib/main.dart [hunk 4]
```diff
@@ -72,19 +72,17 @@ class ForDemocracyApp extends StatelessWidget {
...
       theme: ThemeData(
-        primaryColor: ThemeColors.primary,
-        secondaryHeaderColor: ThemeColors.secondary,
-        colorScheme: ColorScheme.fromSwatch(
-          primarySwatch: ThemeColors.primaryMaterial,
-          accentColor: ThemeColors.secondary,
+        colorScheme: ColorScheme.dark(
+          primary: ThemeColors.primary,
+          secondary: ThemeColors.primary,
+          // background: ThemeColors.surface,
```
Comment (YummYume) [inline mobile/lib/main.dart]:
```suggestion
```

File: mobile/lib/screens/events_screen.dart [hunk 5]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+        builder: (context, snapshot) {
+          if (snapshot.connectionState == ConnectionState.waiting) {
+            // Loading state
+            return Center(
+              child: CircularProgressIndicator(
+                semanticsLabel:
+                    AppLocalizations.of(context)!.planetsScreenLoading,
+              ),
+            );
+          } else if (snapshot.hasError) {
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]:
```suggestion
          }
          
          if (snapshot.hasError || !snapshot.hasData) {
```

File: mobile/lib/screens/events_screen.dart [hunk 6]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+                    style: const TextStyle(color: Colors.red),
+                  ),
+                  TextButton(
+                    onPressed: () => fetchEvents(),
+                    child: Text(AppLocalizations.of(context)!.retry),
+                  ),
+                ],
+              ),
+            );
+          } else {
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]: Retirer le else

File: mobile/lib/screens/events_screen.dart [hunk 7]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+            // Success state
+            final events = snapshot.data;
+
+            if (events == null) {
+              return const Center();
+            }
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]:
```suggestion
```

File: mobile/lib/screens/events_screen.dart [hunk 8]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+                  TextButton(
+                    onPressed: () => fetchEvents(),
+                    child: Text(AppLocalizations.of(context)!.retry),
+                  ),
+                ],
+              ),
+            );
+          } else {
+            // Success state
+            final events = snapshot.data;
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]:
```suggestion
            final events = snapshot.data!;
```

File: mobile/lib/screens/events_screen.dart [hunk 9]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+                return Padding(
+                  padding: const EdgeInsets.only(
+                    left: 8,
+                    right: 8,
+                  ),
+                  child: Column(children: [
+                    // Defences
+                    ListTile(
+                      title: Text('Ongoing defenses',
+                          style: Theme.of(context).textTheme.headlineLarge),
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]:
```suggestion
                      title: Text('Ongoing defenses',
                          style: Theme.of(context).textTheme.headlineLarge,),
```

+ Traduction

File: mobile/lib/screens/events_screen.dart [hunk 10]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+                    ),
+                    ...events.defences.map(
+                      (defence) => ListTile(
+                        title: Text(defence.planet?.name ?? ''),
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]:
```suggestion
                        title: Text(defence.planet.name),
```

File: mobile/lib/screens/events_screen.dart [hunk 11]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+                      ),
+                    ),
+                    // Liberations
+                    ListTile(
+                      title: Text('Ongoing liberations',
+                          style: Theme.of(context).textTheme.headlineLarge),
+                    ),
+                    ...events.liberations.map(
+                      (liberation) => ListTile(
+                        title: Text(liberation.planet?.name ?? ''),
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]:
```suggestion
                        title: Text(liberation.planet.name),
```

File: mobile/lib/screens/events_screen.dart [hunk 12]
```diff
@@ -12,10 +13,95 @@ class EventsScreen extends StatefulWidget {
...
+                    ),
+                    ...events.defences.map(
+                      (defence) => ListTile(
+                        title: Text(defence.planet?.name ?? ''),
+                      ),
+                    ),
+                    // Liberations
+                    ListTile(
+                      title: Text('Ongoing liberations',
+                          style: Theme.of(context).textTheme.headlineLarge),
```
Comment (YummYume) [inline mobile/lib/screens/events_screen.dart]: Pareil

File: mobile/lib/widgets/event/detail.dart [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(16),
+      child: Column(
+        children: [
+          Row(
+            children: [
+              const Text('Some text'),
```
Comment (YummYume) [inline mobile/lib/widgets/event/detail.dart]: Mettre un TODO si pas terminé ?

File: mobile/lib/widgets/event/detail.dart [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+              const Spacer(),
+              GestureDetector(
+                child: const Text(
+                  'view planet',
+                  style: TextStyle(
+                      // color: Colors.blue,
```
Comment (YummYume) [inline mobile/lib/widgets/event/detail.dart]:
```suggestion
```

**Review by YummYume — CHANGES_REQUESTED** [review]

**Review by YummYume — APPROVED** [review]

**Pull request merged**